orttraining/orttraining/core/framework/distributed_run_context.h

```diff
@@ -23,9 +23,19 @@ struct WorkerGroup {
   int32_t rank_in_group{-1};  // current worker' relative rank within this group, ranging from 0 to size-1
 
   std::string ToString() const {
-    return "group_type: " + std::to_string(group_type) + ", group_id: " + std::to_string(group_id) +
+    auto msg = "group_type: " + std::to_string(group_type) + ", group_id: " + std::to_string(group_id) +
```

Comment:
streams are more efficient than std::string to concatenate strings

Reply:
Thanks for the tip!

---
In reply to: [532619031](https://github.com/microsoft/onnxruntime/pull/5815#discussion_r532619031) [](ancestors = 532619031)